packages/lib/src/vis/models.ts

```diff
@@ -87,12 +81,6 @@ export interface AxisParams {
 
 export type Coords = [x: number, y: number];
 
-export type PrintableType =
```

Comment:
Duplicate type that was used by `MatrixVis` (and actually missing `EnumType`) before I removed it in #1702 (I had done that multiple times, but it got lost in my many recent stashes).